docs/angular/slides.md

```diff
@@ -166,7 +166,7 @@ See <a href="https://swiperjs.com/angular#usage" target="_blank" rel="noopener n
 
 ## The IonicSlides Module
 
-With `ion-slides`, Ionic automatically customized dozens of Swiper properties. This resulted in an experience that felt smooth when swiping on mobile devices. We recommend using the `IonicSlides` module to ensure that these properties are also set when using Swiper directly.
+With `ion-slides`, Ionic automatically customized dozens of Swiper properties, for example turning off zoom on doubleclick. This resulted in an experience that felt smooth when swiping on mobile devices. We recommend using the `IonicSlides` module to ensure that these properties are also set when using Swiper directly.
```

Comment:
Instead of using an example, let's mention in the docs to review the provided properties from [IonicSlides.tsx](https://github.com/ionic-team/ionic-framework/blob/9f51bdc14556aed0047627f147defc3d0043046c/core/src/components/slides/IonicSlides.ts#L3). This would allow users to quickly reference what properties are being set through IonicSlides Module.

I would recommend adding it to the end of the paragraph.